client/components/app.jsx

```diff
@@ -29,9 +29,30 @@ class App extends React.Component {
 
     this.state = {
       open: false,
-      clicked: false
+      clicked: false,
+      about: [],
+      minimized: ''
     }
     this.handleInfoClick = this.handleInfoClick.bind(this);
+    this.getRandomNumber = this.getRandomNumber.bind(this);
+  }
+
+  componentDidMount(){
+    let id = parseInt(window.location.pathname.slice(1, window.location.pathname.length-1))
+    if (isNaN(id)) {
+      id = this.getRandomNumber(1, 100);
+    }
+
+     axios
+     .get(`/api/about:${id}`)
+      .then(response => {
+        var split = response.data.description.split('.');
+        var min = split[0] + '. ' + split[1] + '.';
+        this.setState({
+          about: response.data,
+          minimized: min
+        });
```

Comment:
if you rename `var min` to `var minimized` instead, you can use es6 syntax to turn this chunk of code to
```
        this.setState({
          about: response.data,
          minimized
        });
```